### PR TITLE
ENS Profile zoomable images

### PR DESCRIPTION
**Pull request by jxom**
Fixes RNBW-3199

## What changed (plus any additional context for devs)

This PR implements a more flexible `ImagePreviewOverlay` component that can be used for multiple images that require zoomable functionality.

Mainly, this is implemented on the ENS profiles screen.

## PoW (screenshots / screen recordings)

https://www.loom.com/share/838126b300cf4a4da1079f639032ae1b

### CODE REVIEW

File: src/components/expanded-state/unique-token/ZoomableWrapper.android.js [hunk 1]
```diff
@@ -83,7 +85,8 @@ export const ZoomableWrapper = ({
   height,
 }) => {
   // eslint-disable-next-line react-hooks/rules-of-hooks
-  const animationProgress = givenAnimationProgress || useSharedValue(0);
```
Comment (osdnk) [inline src/components/expanded-state/unique-token/ZoomableWrapper.android.js:86]: what's the difference between global and non-global?

Comment (jxom) [reply src/components/expanded-state/unique-token/ZoomableWrapper.android.js]: lol there is none 🤣 not sure why i did that.
removed the dupe.

**Comment by estebanmino** [comment]
i noticed some issues trying this PR
https://www.loom.com/share/63be7cec92ed433ba1ffbe11b43bb2f1

iOS
- swipe down on PFP would not only close the PFP, it's closing the profiles sheet

Android
- if there's no cover the background is not taking the correct accent color, in the video the background cover is red even tho the background in ios is brown
- there's some square behind the PFP  
image
 - i can press everything behind the expanded image, like opening NFT collections or specific NFTs
 - when I zoom in / zoom out the center of the action is not the center of the two fingers pressing the screen as ios https://recordit.co/c5Ep2h2EoW here I'm doing the zoom gesture in the middle of the screen but you can notice is moving right to left while is zooming in / out

**Review by estebanmino — APPROVED** [review]
🐐

**Pull request merged**